docker-compose.yaml

```diff
@@ -1,27 +1,30 @@
 version: "2.2"
 services:
   api:
-    image: camptocamp/swissgeol_local_api:latest
-    init: true # handle kill signals for rust
+    build:
+      context: ./api
+      dockerfile: Dockerfile
+      args:
+        - ENV=development
+      target: builder
```

Comment:
It feels interesting to reuse the same Dockerfile for release and for local dev.
But, what about:
- the speed: using release mode takes much longer to compile
- can we compile dependencies in release mode but api in debug mode (like you do here)?
- the debugger experience, is it the same in release / debug mode (from my experience of C/C++ release mode have deep optimiztion making the code flow sometime impossible to follow step by step)?

Reply:
Release mode is only used when building all stages of the image, for dev only the first one is built then we call watch which uses debug build. Release & debug builds do not share cache.